### PR TITLE
Dev ulla

### PR DESCRIPTION
**Pull request by ullataponen**
- Add styling to Work Experience & Education section and some general styling (headers, containers)
- Move Skills to Resume page and style section

### CODE REVIEW

File: dist/stylesheets/reset.css [hunk 1]
```diff
@@ -1,36 +1,45 @@
 html {
-  box-sizing: border-box; }
```
Comment (evan-kapantais) [inline dist/stylesheets/reset.css:2]: The files in the dist folder should not be edited directly.
Adding additional lines to auto-compiled files also adds to their size so please revert this.

File: dist/stylesheets/style.css [hunk 2]
```diff
@@ -14,45 +16,135 @@ nav {
   justify-content: space-between;
   padding: 20px;
   background: #ffa9a3;
-  transition: 300ms; }
```
Comment (evan-kapantais) [inline dist/stylesheets/style.css:17]: All this should be auto-generated when saving the src/stylesheets/style.scss file. Do it like this.

File: package.json [hunk 3]
```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
+    "scss": "node-sass --watch src/stylesheets - dist/stylesheets"
```
Comment (evan-kapantais) [inline package.json]: This command is wrong. We should remove and not fiddle with this.

If you have sass installed globally you could run sass --watch src/stylesheets:dist/stylesheets

File: src/stylesheets/reset.css [hunk 4]
```diff
@@ -0,0 +1,46 @@
+html {
```
Comment (evan-kapantais) [inline src/stylesheets/reset.css]: This should be an scss file, not css. Otherwise we are recompiling it into css for no reason.

**Review by evan-kapantais — CHANGES_REQUESTED** [review]
Look at the comments I have left in the files.

File: dist/stylesheets/reset.css (hunk 1 above)
Comment (ullataponen) [reply dist/stylesheets/reset.css:2]: My reset.css file has that row, so not sure why it shows up here. Pls check e8f95252e6126079cd7e122d5778a952bcbf9177.

File: dist/stylesheets/style.css (hunk 2 above)
Comment (ullataponen) [reply dist/stylesheets/style.css:17]: My style.css file has that row, pls check 900af5f3f31e88662b4ab0d62298569fa5787fed.

**Review by evan-kapantais — APPROVED** [review]

**Pull request merged**